cumulusci/utils/yaml/safer_loader.py

```diff
@@ -16,15 +16,11 @@
 
 pattern = re.compile(r"^\s*[\u00A0]+\s*", re.MULTILINE)
 
-logger = getLogger(__name__)
+default_logger = getLogger(__name__)
 
 
-def _replace_nbsp(origdata):
-    """Replace non-breaking spaces with spaces.txt
-
-    They sometimes arise in copy and paste from Trailhead or
-    elsewhere and are hard to debug."""
-
+def _replace_nbsp(origdata, filename, logger=default_logger):
```

Comment:
I think we probably can always use the default logger. Generally when we want to redirect log output we do that by changing the handlers attached to the "cumulusci" logger (which also processes log entries from other loggers starting with `cumulusci.`)